sonar-plugin/src/main/java/io/ecocode/javascript/ESLintRulesBundle.java

```diff
@@ -0,0 +1,12 @@
+package io.ecocode.javascript;
+
+import org.sonar.plugins.javascript.api.RulesBundle;
+
+public class ESLintRulesBundle implements RulesBundle {
```

Comment:
class really needed, only to get a tgz file name ?

Reply:
Yes I need it to link the Sonar plugin with the ESLint plugin :+1: